core.yaml

```diff
@@ -10,6 +10,8 @@ k8s:
     - name: istio-system
       disableIstioInjection: true
     - name: knative-serving
+    - name: kubeapps
+      disableIstioInjection: true
```

Comment:
Can we make it part of mesh?

Reply:
No, and I think we don't want to. I think we should limit the scope of istio proxy usage to only include sensitive workloads, and not for low level automation. The operators install apps into namespaces we make available for the admins/teams to deploy into, which will have autoinjection.